### PR TITLE
removed extra words

### PR DESCRIPTION
**Pull request by muwanguzif**
in this PR :
removed extra words.
added spacing

### CODE REVIEW

**Comment by muwanguzif** [comment]
fixed the image positioning

**Comment by powermukisa** [comment]
Great work! I'll let Rich review and approve

File: style.css [hunk 1]
```diff
@@ -71,24 +71,48 @@ header {
...
 
-.features, #container1, #container2, #container3 {
+.features{
     background: rgb(140, 140, 184);  
     border-radius: 50px;
 }
 
-#txt1, #txt2, #txt3 {
-    font-size: 20px;
-    margin-top: -60px;
```
Comment (RichKitibwa) [inline style.css:91]: The text describing the features isn't centered horizontally and vertically. You need to center the text in th div for a better look

File: style.css [hunk 2]
```diff
@@ -71,24 +71,48 @@ header {
...
+    
+}
+
+#container3 {
+    display: flex;
+    margin-top: 40px;
+    background: #264570;
+    margin-bottom: 40px;
+    border: 3px black solid;
+
```
Comment (RichKitibwa) [inline style.css:120]: The colors need to be uniform for the background of the features, it gives the page a better look. Also you need to consider lighter colors like light shades of gray

**Review by RichKitibwa — COMMENTED** [review]
Good work on placing the images and text in the features section. However, more work needs to be done to improve the color choice and text placement. You used about 5 different colors which make the website look a bit dis organised. 
Check other educational websites to see what common colors they use.

File: style.css (hunk 1 above)
Comment (RichKitibwa) [reply style.css:91]: Read about flex. It makes centering items very easy.

**Review by RichKitibwa — APPROVED** [review]
Generally looks better. The color selection is still dull. However, you didn't exploit Flex fully because I still see the images and text not correctly placed.

**Pull request merged**